appgate/appgate.py

```diff
@@ -306,6 +307,13 @@ async def main_loop(queue: Queue, ctx: Context, k8s_configmap_client: K8SConfigM
                                                         if appgate_client else {},
                                                         k8s_configmap_client=k8s_configmap_client,
                                                         api_spec=ctx.api_spec)
+                    maybe_errors = filter(None, map(lambda p: p.errors, new_plan.entities_plan.values()))
+                    errors = filter(lambda s: len(s) > 0, itertools.chain.from_iterable(maybe_errors))
+                    if errors:
```

Comment:
filter returns an iterable, you need to convert it to a list first if you want to check it with if.

Reply:
ahh yes, i forgot that, why mypy did not complain though?